rules-reviewed/eap7/eap6/hsearch.windup.xml

```diff
@@ -825,7 +825,7 @@ fulltextQuery.setSort(new Sort(new SortField("title", SortField.Type.STRING)))
                 <javaclass references="org.apache.lucene.queryParser.{type}" />
             </when>
             <perform>
-                <hint title="Renamed Lucene class {type} in Lucene 4.x" effort="1" severity="mandatory">
+                <hint title="Package name changed for Lucene class {type} in Lucene 4.x" effort="1" severity="mandatory">
```

Comment:
The other rules for package changes in Lucene say "Renamed package for KeywordAnalyzer in Lucene 4.x". I think this one could also read "Renamed package for {type} in Lucene 4.x".